internal/proto/controller/servers/v1/servers.proto

```diff
@@ -43,3 +43,43 @@ message Server {
 message TagValues {
   repeated string values = 10;
 }
+
+message ServerController {
```

Comment:
I can't name it WorkerStatus because it conflicts with another message name in worker.proto. Do you have preferences for a message name other than ServerWorkerStatus? Otherwise I will keep it as is